graphtools/api.py

```diff
@@ -54,13 +55,20 @@ def Graph(data,
     data : array-like, shape=[n_samples,n_features]
         accepted types: `numpy.ndarray`, `scipy.sparse.spmatrix`.
         TODO: accept pandas dataframes
-
-    n_pca : `int` or `None`, optional (default: `None`)
+    n_pca : `int`, `None`, `False`,'True', 'adaptive', optional
```

Comment:
None/False and True/'adaptive' are redundant, so we could leave them out of the first line description. Maybe {`int`, `None`, 'adaptive'}, optional (default: `None`).